diff --git a/arch/arm64/boot/dts/amlogic/meson64_odroidn2.dts b/arch/arm64/boot/dts/amlogic/meson64_odroidn2.dts
index 4ece5798..1cf9f46d 100644
--- a/arch/arm64/boot/dts/amlogic/meson64_odroidn2.dts
+++ b/arch/arm64/boot/dts/amlogic/meson64_odroidn2.dts
@@ -71,12 +71,12 @@
 			alignment = <0x0 0x400000>;
 			alloc-ranges = <0x0 0x7f800000 0x0 0x800000>;
 		};
-		ion_cma_reserved:linux,ion-dev {
-			compatible = "shared-dma-pool";
-			reusable;
-			size = <0x0 0x8000000>;
-			alignment = <0x0 0x400000>;
-		};
+		//ion_cma_reserved:linux,ion-dev {
+		//	compatible = "shared-dma-pool";
+		//	reusable;
+		//	size = <0x0 0x8000000>;
+		//	alignment = <0x0 0x400000>;
+		//};
 
 		//di_reserved:linux,di {
 			//compatible = "amlogic, di-mem";
@@ -89,28 +89,28 @@
 			//no-map;
 		//};
 		/*di CMA pool */
-		di_cma_reserved:linux,di_cma {
-			compatible = "shared-dma-pool";
-			reusable;
-			/* buffer_size = 3621952(yuv422 8bit)
-			 *  | 4736064(yuv422 10bit)
-			 *  | 4074560(yuv422 10bit full pack mode)
-			 * 10x3621952=34.6M(0x23) support 8bit
-			 * 10x4736064=45.2M(0x2e) support 12bit
-			 * 10x4074560=40M(0x28) support 10bit
-			 */
-			size = <0x0 0x02800000>;
-			alignment = <0x0 0x400000>;
-		};
-		codec_mm_cma:linux,codec_mm_cma {
-			compatible = "shared-dma-pool";
-			reusable;
-			/* ion_codec_mm max can alloc size 80M*/
-			size = <0x0 0x13400000>;
-			alignment = <0x0 0x400000>;
-			linux,contiguous-region;
-			clear-map;
-		};
+		//di_cma_reserved:linux,di_cma {
+		//	compatible = "shared-dma-pool";
+		//	reusable;
+		//	/* buffer_size = 3621952(yuv422 8bit)
+		//	 *  | 4736064(yuv422 10bit)
+		//	 *  | 4074560(yuv422 10bit full pack mode)
+		//	 * 10x3621952=34.6M(0x23) support 8bit
+		//	 * 10x4736064=45.2M(0x2e) support 12bit
+		//	 * 10x4074560=40M(0x28) support 10bit
+		//	 */
+		//	size = <0x0 0x02800000>;
+		//	alignment = <0x0 0x400000>;
+		//};
+		//codec_mm_cma:linux,codec_mm_cma {
+		//	compatible = "shared-dma-pool";
+		//	reusable;
+		//	/* ion_codec_mm max can alloc size 80M*/
+		//	size = <0x0 0x13400000>;
+		//	alignment = <0x0 0x400000>;
+		//	linux,contiguous-region;
+		//	clear-map;
+		//};
 		/* codec shared reserved */
 		codec_mm_reserved:linux,codec_mm_reserved {
 			compatible = "amlogic, codec-mm-reserved";
@@ -191,7 +191,7 @@
 		/* 0:use reserved; 1:use cma; 2:use cma as reserved */
 		flag_cma = <1>;
 		//memory-region = <&di_reserved>;
-		memory-region = <&di_cma_reserved>;
+	//	memory-region = <&di_cma_reserved>;
 		interrupts = <0 46 1
 				0 40 1>;
 		interrupt-names = "pre_irq", "post_irq";
diff --git a/arch/arm64/boot/dts/amlogic/mesong12_odroid_common.dtsi b/arch/arm64/boot/dts/amlogic/mesong12_odroid_common.dtsi
index 54b4075f..d1b94e63 100644
--- a/arch/arm64/boot/dts/amlogic/mesong12_odroid_common.dtsi
+++ b/arch/arm64/boot/dts/amlogic/mesong12_odroid_common.dtsi
@@ -41,7 +41,7 @@
 
 	codec_mm {
 		compatible = "amlogic, codec, mm";
-		memory-region = <&codec_mm_cma &codec_mm_reserved>;
+	//	memory-region = <&codec_mm_cma &codec_mm_reserved>;
 		dev_name = "codec_mm";
 		status = "okay";
 	};
diff --git a/arch/arm64/boot/dts/amlogic/mesong12b.dtsi b/arch/arm64/boot/dts/amlogic/mesong12b.dtsi
index 3786fbdf..d6155958 100644
--- a/arch/arm64/boot/dts/amlogic/mesong12b.dtsi
+++ b/arch/arm64/boot/dts/amlogic/mesong12b.dtsi
@@ -862,7 +862,7 @@
 
 		ion_dev {
 			compatible = "amlogic, ion_dev";
-			memory-region = <&ion_cma_reserved>;
+		//	memory-region = <&ion_cma_reserved>;
 		};/* end of ion_dev*/
 
 		audiobus: audiobus@0xff642000 {
diff --git a/arch/arm64/boot/dts/amlogic/mesong12b_a.dtsi b/arch/arm64/boot/dts/amlogic/mesong12b_a.dtsi
index 03afb237..a03572b9 100644
--- a/arch/arm64/boot/dts/amlogic/mesong12b_a.dtsi
+++ b/arch/arm64/boot/dts/amlogic/mesong12b_a.dtsi
@@ -837,7 +837,7 @@
 
 		ion_dev {
 			compatible = "amlogic, ion_dev";
-			memory-region = <&ion_cma_reserved>;
+		//	memory-region = <&ion_cma_reserved>;
 		};/* end of ion_dev*/
 
 		audiobus: audiobus@0xff642000 {
